src/server/game/Spells/SpellEffects.cpp

```diff
@@ -273,6 +276,15 @@ void Spell::EffectResurrectNew(SpellEffIndex effIndex)
     SendResurrectRequest(target);
 }
 
+void DelayMe(float seconds)
+{
+    clock_t startClock = clock();
+    float secondsAhead = seconds * CLOCKS_PER_SEC;
+    // do nothing until the elapsed time has passed.
+    while (clock() < startClock + secondsAhead);
```

Comment:
```suggestion
    std::clock_t startClock = std::clock();
    float secondsAhead = seconds * CLOCKS_PER_SEC;
    // do nothing until the elapsed time has passed.
    while (std::clock() < startClock + secondsAhead);
```
If I'm not missing something :P

Reply:
D: gasp. addded in